dotCMS/src/integration-test/java/com/dotcms/enterprise/rules/RulesAPIImplIntegrationTest.java

```diff
@@ -144,13 +217,78 @@ public void shouldSaveRuleAction() throws DotSecurityException, DotDataException
         final Host host = new SiteDataGen().nextPersisted();
         final Rule rule = new RuleDataGen().host(host).nextPersisted();
 
-        this.addPermission(role, host, true);
+        addRulesReadPermissions(role, host);
+        addRulesPublishPermissions(role, host);
 
         final RuleAction ruleAction = new RuleActionDataGen().rule(rule).next();
 
         rulesAPI.saveRuleAction(ruleAction, user, false);
 
         final List<Rule> allRules = rulesAPI.getAllRules(user, false);
+        checkActions(ruleAction, allRules);
+    }
+
+    /**
+     * Method to Test: {@link RulesAPIImpl#saveRuleAction(RuleAction, User, boolean)}
+     * When: User with permission try to create a Rule's Action in a Rule's Page
+     * Should: Save it
+     *
+     * @throws DotSecurityException
+     * @throws DotDataException
+     */
+    @Test
+    public void shouldSaveRuleActionInPage() throws DotSecurityException, DotDataException {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [JUnit tests should include assert() or fail()](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5259424)